weblab/experiments/apps.py

```diff
@@ -8,7 +8,7 @@ class ExperimentsConfig(AppConfig):
     name = 'experiments'
 
     def ready(self):
-        from .models import ExperimentVersion, RunningExperiment
+        from .models import Runnable, RunningExperiment
```

Comment:
suspect this is wrong and I need to still reference ExperimentVersion here, but not sure why

Reply:
No, I think this one wants to be `Runnable` - it's making sure the results on disk are removed when the DB entry is deleted. And the results on disk are associated with `Runnable` now.